server/utils/checkYourAnswersUtils.test.ts

```diff
@@ -17,7 +17,7 @@ const {
   getAnswer,
   summaryListItemForQuestion,
   checkYourAnswersSections,
-  getSectionsWithAnswers,
+  getSections,
```

Comment:
interesting! I think we were trying for in `getSectionsWithAnswers` was a name which would communicate that we only want sections _which have answers_ rather than _sections and their answers_! i.e. sections without any answers (e.g. 'check your answers') are omitted from what's returned. Maybe `getSections()` is adequate rather than something more pedantic like `getSectionsContainingQuestions()`?